src/pages/Registration/Steps/WalletRegistration/WalletSubmission.tsx

```diff
@@ -20,7 +20,7 @@ export default function WalletSubmission({
   const { isSubmitting, registerWallet } = useRegisterWallet();
   const { data } = useRegState<3>();
 
-  if (providerId !== "web3auth-torus") {
+  if (!(providerId === "web3auth-torus" || providerId === "metamask")) {
```

Comment:
fixes metamask tagged us `invalid wallet` upon connection

Reply:
Nit: Maybe we can clarify the check's intent a bit here?
```suggestion
  if (!["web3auth-torus",  "metamask"].includes(providerId)) {
```

Reply:
comparing directly with `providerId` is safer as it provides auto completion and warns if compared against a value that's not `type ProviderId`

Reply:
> comparing directly with `providerId` is safer as it provides auto completion and warns if compared against a value that's not `type ProviderId`

That's fair. :+1: